app/src/Contacts.js

```diff
@@ -0,0 +1,138 @@
+import * as React from "react";
+
+import * as apiClient from "./apiClient";
+import "./global.module.scss";
+import Card from "./ui/Card";
+
+const Contacts = () => {
+  const [contacts, setContacts] = React.useState([]);
+  const [isAdding, setIsAdding] = React.useState(false);
+
+  const loadContacts = async () => setContacts(await apiClient.getContacts());
+  const addContact = (contact) =>
+    apiClient.addContact(contact).then(loadContacts).then(setIsAdding(false));
+
+  React.useEffect(() => {
+    loadContacts();
+  }, []);
+
+  return (
+    <section>
+      <ContactList contacts={contacts} loadContacts={loadContacts} />
+      {!isAdding ? (
+        <button onClick={() => setIsAdding(!isAdding)}>Add a contact</button>
+      ) : (
+        <>
+          <div className="addContact">
+            <Form action={addContact} contact={{}} button={"Add"} />
+          </div>
+          <button onClick={() => setIsAdding(!isAdding)}>Cancel</button>
+        </>
+      )}
+    </section>
+  );
+};
+
+const ContactCard = ({ contact, loadContacts }) => {
+  const [isEditing, setIsEditing] = React.useState(false);
+
+  const updateContact = (contact, id) => {
+    console.log(contact);
+    setIsEditing(!isEditing);
+    apiClient.updateContact(contact, id).then(loadContacts);
+  };
+
+  return (
+    <li>
+      <Card>
+        <details>
+          <summary>
+            <img src={contact.photo} alt={contact.name} />
```

Comment:
[ux] Since you're relying on `contact.photo` as your summary, and it's an optional field, you want to consider some kind of fallback if it doesn't exist.